src/app/log.py

```diff
@@ -14,6 +14,7 @@ def configure_logging():
     """Configure logging; mozlog format for Dockerflow"""
     logging_config = {
         "version": 1,
+        "disable_existing_loggers": False,
```

Comment:
We want to disable existing loggers, otherwise we don't control what loggers are configured out there

Reply:
I thought the general advice here was to set this to `False` and manually / explicitly handle any loggers?

Reply:
Saw your notes, yes you're right

Reply:
For posterity, these were some sources that guided me to set this to `False`:
- https://guicommits.com/how-to-log-in-python-like-a-pro/
> I recommend keeping disable_existing_loggers False to avoid your system from swallowing other unexpected issues that might happen. If you want to modify some other loggers I recommend (although boring) overwriting them explicitly!
- https://docs.djangoproject.com/en/4.0/topics/logging/#configuring-logging
> Thus you should be very careful using 'disable_existing_loggers': True; it’s probably not what you want. Instead, you can set disable_existing_loggers to False and redefine some or all of the default loggers